src/validator/CreditCardChecking.java

```diff
@@ -0,0 +1,5 @@
+package validator;
+
+public interface CreditCardChecking {
+	public boolean checkValidity(CreditCardVendor ccv, String number);
```

Comment:
consider adding final modifier as both object should not be modified inside this process